src/js/popup.js

```diff
@@ -30,27 +34,63 @@ function showSignUpPanel() {
   return signUpOrInPanel.classList.remove("hidden");
 }
 
+function choosePanel(numRemaining, panelId, premium){
+  if (premium){
+    document.getElementsByClassName("content-wrapper")[0].remove();
+    return 'premiumPanel';
+  }
+  else {
+    document.getElementsByClassName("premium-wrapper")[0].remove();
+    return (numRemaining === 0) ? "maxAliasesPanel" : `panel${panelId}`
+  }
+}
 
 async function showRelayPanel(tipPanelToShow) {
+  const premiumPanelWrapper = document.querySelector(".premium-wrapper");
   const onboardingPanelWrapper = document.querySelector("onboarding-panel");
   const tipImageEl = onboardingPanelWrapper.querySelector("img");
   const tipHeadlineEl = onboardingPanelWrapper.querySelector("h1");
   const tipBodyEl = onboardingPanelWrapper.querySelector("p");
   const currentPanel = onboardingPanelWrapper.querySelector(".current-panel");
   const onboardingPanelStrings = getOnboardingPanels();
 
+  const aliasesUsedValEl = premiumPanelWrapper.querySelector(".aliases-used");
+  const emailsBlockedValEl = premiumPanelWrapper.querySelector(".emails-forwarded");
+  const emailsForwardedValEl = premiumPanelWrapper.querySelector(".emails-blocked");
```

Comment:
typo? are these accidentally reversed?
```suggestion
  const emailsBlockedValEl = premiumPanelWrapper.querySelector(".emails-blocked");
  const emailsForwardedValEl = premiumPanelWrapper.querySelector(".emails-forwarded");
```

<img width="862" alt="image" src="https://user-images.githubusercontent.com/71928/130639664-b7cafe60-0945-4f3f-b3b5-fa62fdb28903.png">

Reply:
Oh shoot good catch!